src/comm/Connection.cc

```diff
@@ -180,6 +178,21 @@ Comm::Connection::connectTimeout(const time_t fwdStart) const
     return min(ctimeout, ftimeout);
 }
 
+void
+Comm::Connection::updateIdent(const Ident::Lookup &newLookup)
+{
+    // Multiple updateIdent() calls happen when a configuration directive (that
+    // requires a lookup) is evaluated while a lookup attempt is still pending.
+    // Ident::IdentStateData::notify() makes calls with the same lookup value.
```

Comment:
The comment is not clear - probably a typo but worth fixing

Reply:
> The comment is not clear - probably a typo but worth fixing

@kinkie, please identify what is not clear or where you see a probable typo. I do not see any particular problems in this comment, so I cannot suggest what to improve.

Reply:
@kinkie, thank you for answering my question during the meeting. I rephrased this comment in branch commit e6f9c78ea4d918e1712c401856dcb125de607f21. Please see whether that commit addresses your concern.

BTW, these repeated calls should be eliminated, but that is a separate/old problem that deserves a dedicated PR.